src/data/posts.ts

```diff
@@ -15,57 +15,944 @@ class Post {
 }
 
 export const POST = {
-  ms2024: new Post({
+  kixlab2016: new Post({
```

Comment:
wouldn't it be easier to add items if the order is reversed? (i.e., adding the newest post at the top.)